lib/csu/common-cheri/crt_init_globals.c

```diff
@@ -74,7 +75,7 @@ crt_init_globals(void)
 }
 #endif /* !CRT_INIT_GLOBALS_GDC_ONLY */
 
-#ifndef POSITION_INDEPENDENT_STARTUP
+#ifndef PIC
```

Comment:
Oops. :(

Reply:
Yeah though in practice it doesn't matter, it just means the static inlined function is made available to the including file and then dropped from the output due to not being used.